### PR TITLE
Issue 101 gcp

### LINKED ISSUE #101: Add GCP support

### PR DESCRIPTION
**Pull request by Tomerva**
fix #101

### CODE REVIEW

File: dragoneye/cloud_scanner/gcp/gcp_scanner.py [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+    @elapsed_time('Scanning GCP live environment took {} seconds')
+    def scan(self) -> str:
+        scan_commands = load_yaml(self.settings.commands_path)
+        self.account_data_dir = init_directory(self.settings.output_path, self.settings.account_name, self.settings.clean)
+
+        dependable_commands = []
+        non_dependable_commands = []
+        for command in scan_commands:
+            if "Parameters" in command:
```
Comment (alonashkenazi) [inline dragoneye/cloud_scanner/gcp/gcp_scanner.py:32]: You are using $project so all cmd are dependable, no?

File: dragoneye/cloud_scanner/gcp/gcp_scanner.py [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+        for dependable_command in dependable_commands:
+            dependable_tasks.append(ThreadedFunctionData(
+                self._execute_scan_commands,
+                (dependable_command,),
+                'exception on command {}'.format(dependable_command)))
+
+        for dependable_task in dependable_tasks:
+            deque_tasks.append([dependable_task])
+        execute_parallel_functions_in_threads(deque_tasks, 20)
```
Comment (alonashkenazi) [inline dragoneye/cloud_scanner/gcp/gcp_scanner.py:57]: Should we move the 20 to a config file?

File: dragoneye/cloud_scanner/gcp/gcp_scanner.py (hunk 1 above)
Comment (Tomerva) [reply dragoneye/cloud_scanner/gcp/gcp_scanner.py:32]: $project is a static value, so it doesnt make commands dependable

File: dragoneye/cloud_scanner/gcp/gcp_scanner.py (hunk 2 above)
Comment (Tomerva) [reply dragoneye/cloud_scanner/gcp/gcp_scanner.py]: moved it

**Review by alonashkenazi — APPROVED** [review]

**Pull request merged**